src/olympia/tags/tests/test_views.py

```diff
@@ -0,0 +1,25 @@
+from olympia.amo.tests import APITestClient, TestCase, reverse_ns
+
+from ..models import Tag
+
+
+class TestStaticCategoryView(TestCase):
```

Comment:
Should this be called `TestStaticTagView`?

Reply:
You can see where I copied the `TestCase` from 😬